.gitignore

```diff
@@ -27,3 +27,100 @@ yarn-error.log*
 
 # lighthouse report
 /localhost_*.report.html
+
+# Jetbrains files
+.idea/
```

Comment:
i'm not morally opposed to ignoring editor files in the project gitignore, but since your editor is a personal decision, you could add them to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

Reply:
I can put it in my global gitignore. no biggie.

Reply:
Also, .DS_Store is in there.. i had added it again on accident, but that would also apply to your comment too. That's a OSX specific thing and operating systems to develop are a personal choice too ;).